### ISSUE TITLE
Entry file is not marked as executable.

### ISSUE THREAD

**Issue by rohitlodha** [heartsucker/node-deb#68]
I use 

`"node_deb": {
    "entrypoints": {
      "daemon": "server.js "
    }
  }`

server.js is not marked executable and so install fails.

**Comment by heartsucker** [comment]
Yeah, I mean that'd be a nicety to have it auto change it. I already do a lot of messing around with the files, but I think I could add a `chmod +x` to `node-deb` to fix that. I'm trying to not meddle *too* much with people's files and hope they know to do some of these things. I'll think about this one.

**Comment by rohitlodha** [comment]
This is especially true for windows users like me!
